src/main/java/rx/observables/SwingObservable.java

```diff
@@ -163,6 +167,40 @@ public Boolean call(KeyEvent event) {
     }
 
     /**
+     * Creates an observable corresponding to item events.
+     * 
+     * @param component
+     *            The ItemSelectable to register the observable for.
+     * @return Observable emitting the item events for the given itemSelectable.
+     */
+    public static Observable<ItemEvent> fromItemEvents(ItemSelectable itemSelectable) {
+        return ItemEventSource.fromItemEventsOf( itemSelectable );
+    }
+    
+    /**
+     * Creates an observable corresponding to item selection events.
+     * 
+     * @param component
+     *            The ItemSelectable to register the observable for.
+     * @return Observable emitting the an item event whenever the given itemSelectable is selected.
+     */
+    public static Observable<ItemEvent> fromSelectionEvents(ItemSelectable itemSelectable) {
+        return ItemEventSource.fromItemEventsOf( itemSelectable ).filter( SELECTED );
+    }
+    
+    /**
+     * Creates an observable corresponding to item deselection events.
+     * 
+     * @param component
+     *            The ItemSelectable to register the observable for.
+     * @return Observable emitting the an item event whenever the given itemSelectable is deselected.
+     */
+    public static Observable<ItemEvent> fromDeselectionEvents(ItemSelectable itemSelectable) {
+        return ItemEventSource.fromItemEventsOf( itemSelectable ).filter( SELECTED );
```

Comment:
Okay, let me know, what you decide, I was just following along with the pattern I saw for other Sources. It seems that it would be nice to be able to filter on item event type.